### PR TITLE
Open post endpoint for Project Suggestions

### PR DESCRIPTION
**Pull request by MightyAx**
- [x] Conversion of projects-index suggestion json to ingest project
- [x] New suggestion wrangling status
- [x] security opening

### CODE REVIEW

**Review by jacobwindsor — APPROVED** [review]

File: src/main/java/org/humancellatlas/ingest/project/ProjectService.java [hunk 1]
```diff
@@ -73,6 +73,20 @@ public Project register(final Project project) {
         return persistentProject;
     }
 
+    public Project createSuggestedProject(final ObjectNode suggestion) {
+        Project suggestedProject = new Project(null);
+        suggestedProject.setWranglingState(WranglingState.NEW_SUGGESTION);
+        var notes = String.format(
+            "DOI: %s \nName: %s \nEmail: %s \nComments: %s",
```
Comment (ke4) [inline src/main/java/org/humancellatlas/ingest/project/ProjectService.java:80]: Why are we adding spaces after the string parameters? Are those spaces needed there?

**Review by ke4 — CHANGES_REQUESTED** [review]
Could we add a test for the new endpoint, please?

File: src/main/java/org/humancellatlas/ingest/project/ProjectService.java (hunk 1 above)
Comment (jacobwindsor) [reply src/main/java/org/humancellatlas/ingest/project/ProjectService.java:80]: It's easier to read like this. Not necessary but not harmful and makes the code easier on the eye

**Review by ke4 — APPROVED** [review]
Would be nice to have a test for the new endpoint

**Review by amnonkhen — APPROVED** [review]

**Pull request merged**